### PR TITLE
tmt, tmt-run, tmt-try: add pages

### PR DESCRIPTION
**Pull request by martinhoyer**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):** 1.38.0

### CODE REVIEW

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/tmt.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`tmt init`
+
+- Create a new test with template and link:
+
+`tmt test create --template {{beakerlib}} --link {{verifies:issue#1234}}`
+
+- List available tests, plans, or stories:
+
+`tmt <test|plan|story> ls {{pattern}}`
```
Comment (acuteenvy) [inline pages/linux/tmt.md:21]:
```suggestion
`tmt {{test|plan|story}} ls {{pattern}}`
```

File: pages/linux/tmt-try.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+> See also: `tmt`, `tmt-run`.
+> More information: <https://tmt.readthedocs.io/en/stable/stories/cli.html#try>.
+
+- Quickly experiment with the default provision method (no tests in cwd):
+
+`tmt try`
+
+- Run a test in the current working directory:
+
+`{{cd path/to/test &&}} tmt try`
```
Comment (acuteenvy) [inline pages/linux/tmt-try.md:13]:
```suggestion
`cd {{path/to/test}} && tmt try`
```

File: pages/linux/tmt-run.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# tmt run
+
+> Execute tmt test steps. By default, all steps are run.
+> See also: `tmt`, `tmt-try`.
```
Comment (spageektti) [inline pages/linux/tmt-run.md:4]:
```suggestion
```
We do not use this syntax to reference other subcommands.

File: pages/linux/tmt-try.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# tmt try
+
+> Quickly experiment with tests and environments.
+> See also: `tmt`, `tmt-run`.
```
Comment (spageektti) [inline pages/linux/tmt-try.md:4]:
```suggestion
```
the same as above

File: pages/linux/tmt.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
+> See also: `tmt-run`, `tmt-try`.
```
Comment (spageektti) [inline pages/linux/tmt.md:4]:
```suggestion
> Some subcommands such as `run`, `try`, etc. have their own usage documentation.
```
For subcommand mention we use this syntax:
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md

File: pages/linux/tmt.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+> More information: <https://tmt.readthedocs.io>.
+
+- Look around to see what's available:
```
Comment (spageektti) [inline pages/linux/tmt.md:7]:
```suggestion
- List available tests, plans, and stories:
```

File: pages/linux/tmt.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+`tmt`
+
+- Initialize tmt files/project structure:
+
+`tmt init`
+
+- Create a new test with template and link:
```
Comment (spageektti) [inline pages/linux/tmt.md:15]:
```suggestion
- Create a new test with a template and link:
```

File: pages/linux/tmt.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
```
Comment (spageektti) [inline pages/linux/tmt.md:3]:
```suggestion
> Test Management Tool for creating, running, and debugging tests.
```
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#serial-comma

File: pages/linux/tmt-try.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+> More information: <https://tmt.readthedocs.io/en/stable/stories/cli.html#try>.
+
+- Quickly experiment with the default provision method (no tests in cwd):
```
Comment (spageektti) [inline pages/linux/tmt-try.md:7]:
```suggestion
- Quickly experiment with the default provision method (no tests in the CWD):
```

File: pages/linux/tmt-run.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+
+- Run tests with the provided context:
+
+`tmt run --context {{key=value}} -c {{distro=fedora}}`
+
+- Run tests interactively (debug test code in the middle of a test):
+
+`tmt run --all execute --how {{tmt}} --interactive`
+
+- Use dry mode to see what actions would happen and use highest verbosity:
```
Comment (spageektti) [inline pages/linux/tmt-run.md:35]:
```suggestion
- Use dry mode to see what actions would happen and use the highest verbosity:
```

File: pages/linux/tmt-run.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+
+- Run all test steps for each plan:
+
+`tmt run`
+
+- Run only the discover step to show what tests would be run:
+
+`tmt run discover -v`
+
+- Run all steps, adjust the provision step options:
```
Comment (spageektti) [inline pages/linux/tmt-run.md:15]:
```suggestion
- Run all steps and adjust the provision step options:
```

**Review by spageektti — COMMENTED** [review]
Hi. Thank you so much for your contribution. I left a few suggestions.

File: pages/linux/tmt-run.md (hunk 3 above)
Comment (martinhoyer) [reply pages/linux/tmt-run.md:4]: Thanks for the review!  
So, what syntax would you like us to use?
[contributing doc](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md) mentions `For page descriptions, you can additionally use See also: `command`.`  
The other contributing doc page states:
```
The following methods can be used to reference subcommands:

 You can use See also: `command1`, `command2`. template to reference similar commands, aliases and subcommands.
 ```

Comment (martinhoyer) [reply pages/linux/tmt-run.md:4]: oh I see the suggestion below: `Some subcommands such as `run`, `try`, etc. have their own usage documentation.`

**Comment by martinhoyer** [comment]
> Hi. Thank you so much for your contribution to our project. Welcome to tldr-pages! I left a few suggestions for you.

Thank You very much!

**Comment by idorax** [comment]
Hi @martinhoyer, the Chinese version is ready. For details, please refer to https://github.com/tldr-pages/tldr/pull/14962

**Pull request merged**